Zilon.Core/Zilon.Core.Tests/Tactics/ActUsageHandlerSelectorTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using NUnit.Framework;
+using Zilon.Core.Tactics;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Moq;
+using FluentAssertions;
+using Zilon.Core.Graphs;
+using Zilon.Core.Persons;
+
+namespace Zilon.Core.Tactics.Tests
+{
+    [TestFixture()]
+    public class ActUsageHandlerSelectorTests
+    {
+        [Test()]
+        public void GetHandlerTest()
+        {
+            // ARRANGE
+
+            var testHandlerMock = new Mock<IActUsageHandler>();
+
+            var testHandler = testHandlerMock.Object;
+
+            var selector = new ActUsageHandlerSelector(new IActUsageHandler[] { testHandler });
+
+            var targetObject = new Test1();
+
+            // ACT
+
+            var factHandler = selector.GetHandler(targetObject);
+
+            // ASSERT
+
+            factHandler.Should().Be(testHandler);
+        }
+
+        private sealed class Test1 : IAttackTarget
+        {
+            public IGraphNode Node { get; }
+            public PhysicalSize PhysicalSize { get; }
+
+            public bool CanBeDamaged()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'CanBeDamaged'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5403398)